sidebars.js

```diff
@@ -242,9 +228,20 @@ module.exports = {
         },
         {
             type: "category",
-            label: "Getting Started with the SDK",
+            label: "Geocortex Mobile SDK",
             items: [
-                "mobile/sdk-quick-start",
+                {
+                    type: "category",
+                    label: "Quickstart",
```

Comment:
Hmm i'm not so sure anymore about the _Quickstart_ name. For consistency i'm debating on renaming that project to the _vertigis-mobile-sdk_, which also gives us consistency with Web.

Not sure what to do here - i'm not a big fan of the group label.

Reply:
I'm not a big fan of the group label either... 
But with the upcoming work to restructure the SDK, this will likely change or altogether disappear. I think it could be left for now and tackled as part of that work. Thoughts?